sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -404,6 +470,8 @@ export interface BlobStartCopyFromURLOptions {
    * @memberof BlobStartCopyFromURLOptions
    */
   sourceModifiedAccessConditions?: Models.ModifiedAccessConditions;
+  tier?: BlockBlobTier | PremiumPageBlobTier | string;
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Added.